src/Mvc/Mvc.Core/src/ConsumesAttribute.cs

```diff
@@ -96,6 +97,11 @@ public ConsumesAttribute(Type requestType, string contentType, params string[] o
         /// </summary>
         public MediaTypeCollection ContentTypes { get; set; }
 
+        /// <summary>
+        /// Indicates whether the request body is required or not.
+        /// </summary>
+        public bool IsRequired { get; set; } = true;
```

Comment:
Oh fun, it's `true` by default? I wonder if we should flip it - `IsOptional`.

Reply:
What should the default behavior be for MVC ? I don't remember having that discussion yet.